setup.py

```diff
@@ -17,9 +17,11 @@ def get_contents(filename):
 #Make Home Directory
 if not os.path.exists(HOME_DIR):
     os.makedirs(HOME_DIR)
+    os.chmod(HOME_DIR,0o777)
 
 if not os.path.exists(DEFAULT_PICS_FOLDER):
     os.makedirs(DEFAULT_PICS_FOLDER)
+    os.chmod(DEFAULT_PICS_FOLDER,0o777)
```

Comment:
can we only do this for linux? maybe a wrapper function which only adds a mode if it is on linux?